Makefile

```diff
@@ -1,21 +1,122 @@
-.PHONY: \
-	clean \
-	deploy \
-	func-test \
-	unit-test
-
-clean:
-	@echo "Cleaning previous outputs"
-	rm functests/functests.test
-
-deploy:
-	@echo "Deploying Cincinnati operator"
-	hack/deploy.sh
-
-func-test: deploy
-	@echo "Running functional test suite"
-	hack/functest.sh
-
-unit-test:
```

Comment:
You need to retain this target to get through CI.  Once this PR lands, we can pivot to `make test` in the openshift/release job config.